### DISCUSSION TITLE
Dead after 2 weeks

### DISCUSSION THREAD

**Discussion by Vintanerd** [tjboldt/Apple2-IO-RPi#183] [General]
### What happened?

Hi, after 2 weeks the card is dead: no more message at startup, no boot, when I force a "PR#5" I get the message "CHECK DISK DRIVE".
I tried to move on another slot, to make a new microSD but nothing.
The PiZero is working and accessibile by WiFi.
What's the problem? The Eprom or another chip on the card?

Thanks!
:)

### Hardware Version

Sixth Prototype (lastest classic hardware)

### Firmware Version

0010 (classic hardware)

### Driver Version

002D (add support for RPi Pico hardware)

### Shell Version

800F (initial RPi Pico support)

### Where are you seeing this problem?

Boot

### What computer is this running on?

Apple //e (enhanced / platinum)

**Comment by tjboldt** [comment]
It is possible that something else overwrote part of the firmware. See issue #147

**Comment by tjboldt** [comment]
Can you check the logs? Is the service running? See #175
